src/components/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from 'react';
+import Header from './common/Header';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="container-fluid">
+        
+        <Header
+        loading ={this.props.loading}/>
+        {this.props.children}
+      </div>
+    );
+  }
+}
+App.propTypes={
```

Comment:
Could this be written as 
```
App.propTypes = {
```
Be wary of those white space, they make for cleaner and clearer code